docs/index.rst

```diff
@@ -6,15 +6,15 @@ AiiDA ABC tutorial
 +-----------------+-----------------------------------------------------------------------------------------------------------+
 | Related resources                                                                                                           |
 +=================+===========================================================================================================+
-| python packages | `aiida-core 1.4.3`_, `aiida-quantumespresso 3.2.1`_                                                       |
+| python packages | `aiida-core 1.3.0`_, `aiida-quantumespresso 3.0.0`_                                                       |
 +-----------------+-----------------------------------------------------------------------------------------------------------+
-| codes           | `Quantum ESPRESSO 6.0.0`_                                                                                 |
+| codes           | `Quantum ESPRESSO 6.5`_                                                                                   |
 +-----------------+-----------------------------------------------------------------------------------------------------------+
 
-.. _Quantum Mobile 20.03.1: https://github.com/marvel-nccr/quantum-mobile/releases/tag/20.03.1
-.. _aiida-core 1.4.3: https://pypi.org/project/aiida-core/1.4.3/
-.. _aiida-quantumespresso 3.2.1: https://github.com/aiidateam/aiida-quantumespresso/releases/tag/v3.2.1
-.. _Quantum ESPRESSO 6.0.0: https://github.com/QEF/q-e/releases/tag/qe-6.0.0
+.. _Quantum Mobile 20.03.1: https://quantum-mobile.readthedocs.io/en/latest/releases/desktop/20.03.1.html
```

Comment:
Now that I notice this line, I think we can remove it. We don't show the Quantum Mobile version anywhere, since we use the AiiDAlab cluster. Alternatively, we could also mention a row with the AiiDAlab docker stack version?

https://github.com/aiidalab/aiidalab-docker-stack/releases/tag/v21.02.1